plugins/blip/chartdailyfactory.js

```diff
@@ -123,7 +123,7 @@ function chartDailyFactory(el, options) {
         'light': ' g'
       }])
       .labelBaseline(options.labelBaseline)
-      .legend(['bolus', 'carbs'])
+      .legend(['bolus', 'carbs', 'rescuecarbs'])
```

Comment:
Must check "rescuecarbs" or "Rescuecarbs" for the translation.

Reply:
let's stick t 'rescuecarbs'